documentation/guides/migrating-from-v8-to-v9.md

```diff
@@ -328,6 +328,79 @@ This function has been deprecated, but the definition can be copied and used loc
 | `shadow(layer)`              | `--p-shadow-layer`       |
 | `shadow(transparent)`        | `--p-shadow-transparent` |
 
+#### `skeleton-page-header-layout()`
+
+| Deprecated Mixin                       | Replacement Value                   |
```

Comment:
Adding "Deprecated" to the titles here. In line with future work as part of #5128